lecli/apiutils.py

```diff
@@ -35,103 +60,116 @@ def init_config():
         print "An empty config file created in path %s, please check and configure it. To learn " \
               "how to get necessary api keys, go to this Logentries documentation page: " \
               "https://docs.logentries.com/docs/api-keys" % \
-              config_file_path
+              CONFIG_FILE_PATH
     else:
-        print "Config file exists in the path: " + config_file_path
+        print "Config file exists in the path: " + CONFIG_FILE_PATH
 
     exit(1)
 
 
 def load_config():
-    config_file = os.path.join(user_config_dir(lecli.__name__), 'config.ini')
-    files_read = CONFIG.read(config_file)
+    """
+    Load config from OS specific config path into ConfigParser object.
+    :return:
+    """
+    files_read = CONFIG.read(CONFIG_FILE_PATH)
     if len(files_read) != 1:
-        print "Error: Config file '%s' not found, generating one..." % config_file
+        print "Error: Config file '%s' not found, generating one..." % CONFIG_FILE_PATH
         init_config()
         exit(1)
+        print_config_error_and_exit()
     if not CONFIG.has_section('Auth'):
-        print "Error: Config file '%s' is missing Auth section" % config_file
-        exit(1)
+        print_config_error_and_exit(section='Auth')
 
 
 def get_ro_apikey():
     """
     Get read-only api key from the config file.
     """
 
-    ro_apikey = None
+    section = 'Auth'
+    config_key = 'ro_api_key'
     try:
-        ro_apikey = CONFIG.get('Auth', 'ro_api_key')
-        if len(ro_apikey) != 36:
-            print 'Error: Read-only API Key not of correct length'
+        ro_api_key = CONFIG.get(section, config_key)
+        if len(ro_api_key) != 36:
+            print_config_error_and_exit(section, 'Read-only API key(%s)' % config_key, ro_api_key)
+        else:
+            return ro_api_key
     except ConfigParser.NoOptionError:
-        print 'Error: Read-only API Key not configured in configuration file'
-        exit(1)
-    return ro_apikey
+        print_config_error_and_exit(section, 'Read-only API key(%s)' % config_key)
 
 
 def get_rw_apikey():
     """
     Get read-write api key from the config file.
     """
 
-    rw_apikey = None
+    section = 'Auth'
+    config_key = 'rw_api_key'
     try:
-        rw_apikey = CONFIG.get('Auth', 'rw_api_key')
-        if len(rw_apikey) != 36:
-            print 'Error: Read/Write API Key not of correct length'
+        rw_api_key = CONFIG.get(section, config_key)
+        if len(rw_api_key) != 36:
+            print_config_error_and_exit(section, 'Read/Write API key(%s)' % config_key, rw_api_key)
```

Comment:
Mutliple usage of Auth section caught my attention, so declared it as global. This is clearer I think.